examples/mui-base-with-tailwind-css/README.md

```diff
@@ -1,47 +1,27 @@
+Download the example [or clone the repo](https://github.com/mui/material-ui):
 
-In the project directory, you can run:
+<!-- #default-branch-switch -->
 
-### `npm start`
+```sh
+curl https://codeload.github.com/mui/material-ui/tar.gz/master | tar -xz --strip=2 material-ui-master/examples/mui-base-with-tailwind-css
+cd mui-base-with-tailwind-css
+```
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+Install it and run:
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+```sh
+npm install
+npm start
+```
 
-### `npm test`
+or:
 
-Launches the test runner in interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+<!-- Tailwind CSS requires running command in the terminal, so it does not work in Codesandbox/Stackblitz with the default create react app templates. Because of this in the Codesandbox & Stackblitz links below, it is added via CDN. We will need to update this link when we do updates in the example. -->
 
-### `npm run build`
+[![Edit on StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/edit/github-bsyjuh?file=public%2Findex.html)
 
-Builds the app for production in the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) in the Create React App docs for more information.
-
-### `npm run eject`
-
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
-
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
-
-In its place, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn more
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
+[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/mui-base-with-tailwind-css-42z1n8?file=/public/index.html)
```

Comment:
I think that we should consider
```suggestion
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/mui/material-ui/tree/master/examples/mui-base-with-tailwind-css)
```

so we don't need to maintain this custom codesandbox?